HISTORY.rst

```diff
@@ -7,6 +7,12 @@ Release History
 unreleased
 +++++++++++++++
 
+**IoT Hub updates**
+
+* Addition of read modules command for edge device
+
+  - az iot edge read-modules
```

Comment:
Still read-modules here vs export-modules

Reply:
forgot the history file, thanks will update😥